pinc/LPage.inc

```diff
@@ -467,7 +467,7 @@ class LPage
 
         $error_msg = project_transition($this->projectid, $this->round->project_bad_state, PT_AUTO);
         if ($error_msg) {
-            echo "$error_msg<br>\n";
+            throw new ProjectException($error_msg);
```

Comment:
This will not get caught in report_bad_page.php which only catches 'ProjectPageException', should we change this to catch it?

Reply:
Yes, we should add `ProjectException` to the list of exceptions that `report_bad_page.php` will accept.

Reply:
We should also catch PageNotOwnedException which can be thrown in page_require.